core/primitives/src/views.rs

```diff
@@ -322,6 +322,35 @@ pub struct ValidatorInfo {
     pub is_slashed: bool,
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DebugChunkStatus {
+    pub shard_id: u64,
+    pub chunk_hash: ChunkHash,
+    pub gas_used: u64,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub processing_time_ms: Option<u64>,
```

Comment:
Regarding `Duration` vs `u64`, I think this (JSON RPC layer) is the only place were we need to use `u64`. Everything else should use Duration.

In theory, we can even use `Duration` here, as it is `Serialize`, but, IIRC, it's JSON is rather ugly.

Reply:
ok